scripts/performance-benchmarks/README.md

```diff
@@ -0,0 +1,120 @@
+# Performance Benchmark
```

Comment:
This `README.md` has all of the instructions.

Reply:
Thanks!!